apps/prairielearn/src/ee/lib/ai-grading/ai-grading.ts

```diff
@@ -153,7 +125,7 @@ export async function aiGrade({
           question,
           instance_question,
           urlPrefix,
-          openai,
+          model: embeddingModel,
```

Comment:
nit: now that we have multiple model variables, it would be great to rename the input parameter here to `embeddingModel`.